rosbridge_library/src/rosbridge_library/internal/ros_loader.py

```diff
@@ -77,28 +77,46 @@ def __init__(
 
 
 def get_message_class(typestring: str) -> Any:
-    """Loads the message type specified.
+    """
+    Load the message type specified.
+
+    Throws exceptions on failure.
```

Comment:
I believe in all these cases you can use the `:raises:` docstring, like shown here

https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format